OpenUpSA/wazimap-ng
# Switch to ORM queue and low cost heroku resource types

## Description

Change to dyno types that we'll have to pay for but which actually exist beyond 25 november.

Replace Redis queue with ORM queue so that we don't have to run and pay for Redis. We also wanted to do this to have queued tasks in the DB to be able to show more information about task status to admin users in future.

I've fixed the S3 config so you should be able to upload files and have them process now. We're still limited to 10k rows in the DB instances used here so watch out not to upload too much data. you only need data for the geographies in the system in any case so no point uploading hundreds of rows of data.

## Related Issue

https://wazimap.atlassian.net/browse/WNCM-593

## How to test it locally

upload files.

make sure they process properly.

make sure you can create new datasets, variables, indicators.

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
